### ISSUE TITLE
cats version?

### ISSUE THREAD

**Issue by fommil** [arosien/scalaz-cheatsheets#4]
would be really really good.

**Comment by arosien** [comment]
thanks for the suggestion to get me off my butt!

https://github.com/arosien/cats-cheatsheets

**Comment by fommil** [comment]
Heh... just when I've changed to scalaz and don't care so much about cats anymore :-P

**Comment by arosien** [comment]
Oh, why's that? Anyway thanks for the motivation to get this out.

**Comment by fommil** [comment]
1. Scalaz has more in it and is more principled (or at least is heading that way)
2. Scalaz is stable (with cats 1.0-WTF releases causing havoc with downstream deps, I gave up. I was spending more time rewriting the book / examples than new material)
3. I find the scalaz community to be larger and typically more knowledgeable
4. I've been unnecessarily tone policed on at least 2 Typelevel gitter channels and had my voice silenced/ignored on at least 2 github tickets, plus individuals have told me I'm on a kind of probation for having attended Lambdaconf and not opposing FCoP. Plus a lot of prominent Typelevelers have blocked or muted me on Twitter (same reason) and are not helping promote my book (e.g. with retweets) or helping to review it or the example application. I don't see how it is feasible for me to write a book promoting their flagship product with all that going on, so I switched to the library where I felt welcome and which has a cooperative community. I still have a commercial interest so I will have a place for people to register their interest in a cats variant and if it reaches a financial target I will write it. Basically, I got the backlash I feared from  https://github.com/typelevel/typelevel.github.com/pull/172
